react/package.json

```diff
@@ -53,8 +53,10 @@
         "notistack": "^1.0.5",
         "react-colorscales": "^0.7.3",
         "react-draggable": "^4.4.4",
+        "react-is": "^17.0.2",
         "reactour": "^1.18.3",
```

Comment:
Can `reactour` be removed now, or do we need it as long as plugin projects don't use WLF?

Reply:
As long as we are keeping `WebvizPluginPlaceholder`, we have to keep `reactour`. Can be removed as soon as it is removed.